vspace/src/main/java/edu/asu/diging/vspace/core/data/ModuleLinkRepository.java

```diff
@@ -3,18 +3,23 @@
 import java.util.List;
 
 import org.javers.spring.annotation.JaversSpringDataAuditable;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.stereotype.Repository;
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
 
+import edu.asu.diging.vspace.core.model.IModule;
 import edu.asu.diging.vspace.core.model.impl.ModuleLink;
 import edu.asu.diging.vspace.core.model.IModuleLink;
 
 @Repository
 @JaversSpringDataAuditable
 public interface ModuleLinkRepository extends PagingAndSortingRepository<ModuleLink, String> {
+
     
+    List<ModuleLink> findByModuleId(String moduleId);
```

Comment:
how is this method different from the next one?

Reply:
This method is not different from the next one. so updated the code according to the existed method.